spam/01_spam_tutorial.py

```diff
@@ -492,11 +541,15 @@ def textblob_subjectivity(x):
 # %% [markdown]
 # For text applications, some of the simplest LFs to write are often just keyword lookups.
 # These will often follow the same execution pattern, so we can create a template and use the `resources` parameter to pass in LF-specific keywords.
+# Similar to the [`labeling_function` decorator](https://snorkel.readthedocs.io/en/master/source/snorkel.labeling.lf.html#snorkel.labeling.lf.core.labeling_function),
```

Comment:
Links seems to be broken

Reply:
It should point to redux instead of master (though we'll have to change it when we launch v0.9).
Same for a few other links here.